meta-mender-core/recipes-bsp/u-boot/u-boot-mender-common.inc

```diff
@@ -71,7 +71,13 @@ do_provide_mender_defines() {
         MENDER_UBOOT_STORAGE_DEVICE=`get_uboot_device_from_device ${MENDER_STORAGE_DEVICE}`
     fi
 
-    case "${KERNEL_IMAGETYPE}" in
+    if [ -n "${MENDER_KERNEL_IMAGETYPE_FORCE}" ]; then
```

Comment:
Don't forget to document this [here](https://docs.mender.io/1.3/artifacts/variables).

Reply:
Done: https://github.com/mendersoftware/mender-docs/pull/360